samples/wifi/radio_test/single_domain/README.rst

```diff
@@ -0,0 +1,20 @@
+.. _wifi_radio_test_sd:
+
+Wi-Fi: Wi-Fi BLE Radio Test Sample (Single Domain)
+##################################################
+
+The Wi-Fi BLE Radio Test sample demonstrates how to use the radio test for both Wi-Fi and BLE protocols using a single domain image i.e., both running on the same core (Application core).
+The sample demonstrates how to use the radio test subcommands to configure the radio test parameters and run the radio test.
+
+The Wi-Fi Radio test also supports programming the Factory Information Configuration Registers (FICR) fields defined in the nRF7002 one-time programmable (OTP) memory.
+
+See the subpages for detailed documentation on the sample and its features.
+
+.. toctree::
+   :maxdepth: 1
+   :caption: Subpages:
+
+   sample_description.rst
+   peripheral_radio_test.rst
```

Comment:
If I understand correctly, then peripheral_radio_test.rst, radio_test_subcommands.rst, and ficr.rst already exist under Peripheral samples (https://ncsdoc.z6.web.core.windows.net/PR-19203/nrf/samples/peripheral/radio_test/README.html) and Wi-Fi: Radio test (Multi Domain) (https://ncsdoc.z6.web.core.windows.net/PR-19203/nrf/samples/wifi/radio_test/multi_domain/README.html). 

If yes, then can we mention only the links to these documentation pages with some related text on the https://ncsdoc.z6.web.core.windows.net/PR-19203/nrf/samples/wifi/radio_test/single_domain/README.html page to avoid duplicate documentation and the related maintenance?